config/setup/browse.go

```diff
@@ -240,7 +244,7 @@ th a {
 				<tr>
 					<td>
 						{{if .IsDir}}&#128194;{{else}}&#128196;{{end}}
-						<a href="{{.URL}}">{{.Name}}</a>
+						<a href="{{.URL}}" id="name">{{.Name}}</a>
```

Comment:
Actually, let's make this a class since there will be more than one per page. (don't forget to change the css to `.name`)